layouts/partials/loop-staff.html

```diff
@@ -6,7 +6,7 @@
 					{{- if (fileExists (printf "static/images/people/%s.jpg" .ContentBaseName)) -}}
 						<img src="{{ (printf "images/people/%s.jpg" .ContentBaseName ) | absURL }}" alt="">
 					{{- else -}}
-						<img src="//www.fillmurray.com/96/96" alt="">
+						<img src="{{ (printf "//ui-avatars.com/api/?name=%s" .ContentBaseName) | absURL }}" alt="">
```

Comment:
In this case, `.Title` is the correct name reference, since `.ContentBaseName` doesn't contain a space and would generate the wrong initials.

Just as above, add the right image size and querify the args.

```suggestion
						<img src="//ui-avatars.com/api/?{{ (querify "name" .Title "size" 96) | safeURL }}" alt="">
```